wp-content/plugins/core/src/Integrations/Gravity_Forms/Form_Markup.php

```diff
@@ -51,12 +52,12 @@ public function customize_gf_choice_other( string $choice_markup, array $choice,
 	 * @filter gform_field_css_class
 	 *
 	 * @param string $classes
-	 * @param array  $field
+	 * @param object $field
 	 * @param array  $form
 	 *
 	 * @return string
 	 */
-	public function add_gf_select_field_class( string $classes, array $field, array $form ): string {
+	public function add_gf_select_field_class( string $classes, object $field, array $form ): string {
```

Comment:
@dpellenwood don't forget this one.

Reply:
Thanks @defunctl , Fixed. Can you give this another quick look-see?